imgboard.php

```diff
@@ -19,7 +19,8 @@
 
 */
 
-require "config.php";
+require("config.php");
+require("config.json");
```

Comment:
loader class is just a a lazy bridge so I don't have to go through and replace every constant in the repo.